web/source/keyboards/kmwkeyboards.ts

```diff
@@ -1,4 +1,6 @@
 /// <reference path="../kmwbase.ts" />
+// Includes a promise polyfill (needed for IE)
```

Comment:
We shouldn't need a Promise polyfill as we no longer support IE and min Chrome version is 33.

Reply:
I noticed that too, and @jahorton told me 
> That's not the only reason we have es6-shim. I'm pretty sure